core/src/main/java/org/elasticsearch/search/aggregations/pipeline/bucketmetrics/BucketMetricsPipelineAggregator.java

```diff
@@ -49,6 +49,7 @@
     protected GapPolicy gapPolicy;
 
     public BucketMetricsPipelineAggregator() {
+        // NOCOMMIT remove me
```

Comment:
I think these should be NORELEASE so you don't break the build when this PR is merged?